api/app/tests/apps/signals/test_api_endpoints.py

```diff
@@ -39,6 +39,7 @@ class TestAuthAPIEndpoints(APITestCase):
         '/signals/auth/category/',
         '/signals/auth/location/',
         '/signals/auth/priority/',
+        '/signals/auth/note/',
```

Comment:
Can you add a test for posting a new note with the API as well.

Reply:
done